Task.dart

```diff
@@ -0,0 +1,17 @@
+class task {
+  static int nextid = 0;
+  int? id;
+  String? title;
+  String? description;
+  bool? IsCompleted;
+  
+  task(this.id, this.title, this.description, this.IsCompleted);
+  
+  task.create() {
+    id = nextid++;
+    title = '';
+    description = '';
+    IsCompleted = false;
+  }

```

Comment:
It would be better to change the constructor to the best practice
```suggestion
  task({
    this.id = nextid++, 
    required this.title, 
    required this.description, 
    this.IsCompleted = false
    });
```